modules/nf-core/lofreq/indelqual/tests/main.nf.test

```diff
@@ -0,0 +1,75 @@
+nextflow_process {
+
+    name "Test Process LOFREQ_INDELQUAL"
+    script "../main.nf"
+    process "LOFREQ_INDELQUAL"
+    config "./nextflow.config"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "lofreq"
+    tag "lofreq/indelqual"
+
+    test("sarscov2 - bam") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test', single_end:false ], // meta map
+                    file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/bam/test.paired_end.sorted.bam', checkIfExists: true)
+                    ]
+                input[1] = [
+                    [ id:'ref' ], // meta map
+                    file(params.modules_testdata_base_path + 'genomics/sarscov2/genome/genome.fasta', checkIfExists: true)
+                    ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                {
+                    def bam = file(process.out.bam.get(0).get(1))
+                    assert bam.exists()
+                },
+                { assert path(process.out.versions.get(0)).md5 == "822d596299afbb3ae6db66bfa9b77b3e" }
+            )
+        }
+
+    }
+
+    test("sarscov2 - bam - stub") {
+
+        options "-stub"
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test', single_end:false ], // meta map
+                    file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/bam/test.paired_end.sorted.bam', checkIfExists: true)
+                    ]
+                input[1] = [
+                    [ id:'ref' ], // meta map
+                    file(params.modules_testdata_base_path + 'genomics/sarscov2/genome/genome.fasta', checkIfExists: true)
+                    ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                {
+                    def bam = file(process.out.bam.get(0).get(1))
+                    assert bam.exists()
+                },
+                { assert path(process.out.versions.get(0)).md5 == "822d596299afbb3ae6db66bfa9b77b3e" }
```

Comment:
At least for the stub test it should be as simple as:
{ assert snapshot(process.out).match() }